src/registrar/management/commands/send_domain_invitations.py

```diff
@@ -58,6 +60,12 @@ def handle(self, **options):
             logger.info("done sending emails and updating transition_domains")
         else:
             logger.info("not sending emails")
+            for email_context in self.emails_to_send:
+                logger.info(
+                    "would send email to %s for %s",
+                    email_context["email"],
+                    email_context["domains"],
+                )
```

Comment:
(optional) One thing that popped into my head was that using Nicolles `TerminalColors` class would be neat here (yellow?) for visual distinction, but I don't think that's used elsewhere in this file